packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -178,57 +245,168 @@ const startNewEconomicCommittee = async ({
   // NB: committee must only publish what it intended to be public
   const marshaller = await E(board).getPublishingMarshaller();
 
-  const { creatorFacet } = await E(zoe).startInstance(
-    committee, // aka electorate
+  trace('Starting new EC Committee Instance');
+  const { instance, creatorFacet } = await E(zoe).startInstance(
+    committee,
     {},
     { committeeName, committeeSize },
     {
       storageNode,
       marshaller,
     },
+    'economicCommittee',
   );
+  trace('Started new EC Committee Instance Successfully');
 
-  const newPoserInvitationP = E(creatorFacet).getPoserInvitation();
+  economicCommittee.reset();
+  economicCommittee.resolve(instance);
 
-  // reset because it's already been resolved
   economicCommitteeCreatorFacet.reset();
   economicCommitteeCreatorFacet.resolve(creatorFacet);
-  return newPoserInvitationP;
 };
 harden(startNewEconomicCommittee);
 
-const main = async permittedPowers => {
-  console.log('starting new economic committee:', runConfig);
-  const newElectoratePoser = await startNewEconomicCommittee(permittedPowers);
+const startNewEconCharter = async ({
+  consume: { zoe },
+  produce: { econCharterKit },
+  installation: {
+    consume: { binaryVoteCounter: counterP, econCommitteeCharter: installP },
+  },
+  instance: {
+    produce: { econCommitteeCharter: instanceP },
+  },
+}) => {
+  const [charterInstall, counterInstall] = await Promise.all([
+    installP,
+    counterP,
+  ]);
```

Comment:
There's no need to await the charter installation. `startInstance()` takes an `ERef<Installation>`.